app/views/pages/home.html.erb

```diff
@@ -1,3 +1,23 @@
-<div class="bg-green-300 border-green-600 border-b p-4 m-4 rounded">
-  <h1><%= t('hello') %></h1>
+<div class="flex flex-wrap overflow-hidden h-screen">
```

Comment:
This page isn't responsive. That is, it doesn't look good when you make the browser roughly the size of a phone.

If we merge without making the page more responsive, I think we should create an issue to come back and make it more responsive later. At some screen size, it should switch from a horizontal to vertical layout.

Reply:
for certain, this PR still needs a lot of work, but I figured before I went any further we should check to see if it's going in the right direction for everyone. Further issues can certainly be opened, I'm not a styling expert so any further fixes are quite welcome.